src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +16,37 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
```

Comment:
You don't need to explicitly get the SessionFactory from HibernateUtil here, as it is already being passed to the constructor of ActorDaoImpl. Use the sessionFactory field from the superclass instead.